helm-charts/nginx-ingress/values.schema.json

```diff
@@ -46,13 +46,13 @@
           "type": "object",
           "default": {},
           "title": "The selectorLabels Schema",
-          "$ref": "file://./helm-charts/nginx-ingress/v1.28.0/_definitions.json#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector/properties/matchLabels"
+          "$ref": "https://raw.githubusercontent.com/nginxinc/kubernetes-json-schema/master/v1.28.0/_definitions.json#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector/properties/matchLabels"
```

Comment:
need to reference local files here so this file can be reverted back, also https://github.com/nginxinc/kubernetes-json-schema/tree/master/v1.29.0 is now available and IC is using the same.

Reply:
thanks @vepatel those issues have been addressed